### ISSUE TITLE
Idea: option to display kana as furigana

### ISSUE THREAD

**Issue by picsi** [birtles/rikaichamp#90]
It would be cool to have an option to display the kana as furigana, which would hopefully save some horizontal space.

**Comment by birtles** [comment]
Yeah, I've thought about doing this but I'm concerned it would make reading the kana harder since it might be too small. I might try it when adding romaji support.
